sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -439,6 +447,7 @@ protected void initializeDeploymentPlanManager() {
             deployPlan = deploy.get();
         }
         deploymentPlanManager = new DefaultPlanManager(deployPlan);
+        deploymentPlanManager.getPlan().getStrategy().proceed();
```

Comment:
No.  It's a fix for the fact that all plans are initialized in an interrupted state now.  The deploy plan should not be interrupted by default.  Hmmm.... canary strategy....